lckernel/cad/math/intersect.cpp

```diff
@@ -20,7 +20,7 @@ bool Intersect::operator()(const lc::geo::Vector& v1, const lc::geo::Vector& v2)
 }
 
 bool Intersect::operator()(const lc::geo::Vector& v, const lc::entity::Point& p) {
```

Comment:
All the intersection methods with `Point` has the same body. You should create a method with `geo::Coordinate` and `geo::Base`, so you have to move the `nearestPointOnEntity` method in `geo::Base` (it's only not defined in `Coordinate` and `Area`).